pkg/config/error.go

```diff
@@ -31,3 +31,12 @@ type ErrEmpty struct {
 func (e ErrEmpty) Error() string {
 	return fmt.Sprintf("%s: empty: %s", e.Name, e.Msg)
 }
+
+type KeyNotFoundError struct {
```

Comment:
How will this be used?

Reply:
It's being used in the txm tests, for example [here](https://github.com/smartcontractkit/chainlink-solana/pull/475/files#diff-0c15ff13416d346cba7814e0b6501c118492a504ee2bb0263e77d6b6c10280aaR52).

Reply:
Hmmm OK - we might have to revisit this with plugins, since error types are not preserved over the wire. Fortunately it seems like we could just return nil instead of a custom error for this particular case.